requirements.txt

```diff
@@ -29,5 +29,5 @@ requests-oauthlib==0.8.0
 six==1.10.0
 tzlocal==1.3
 whitenoise==3.3.0
-mozilla-django-oidc==0.1.10
 django-environ==0.4.1
+git+git://github.com/mozilla/mozilla-django-oidc#egg=mozilla-django-oidc
```

Comment:
Shouldn't this be `git+git://github.com/mozilla/mozilla-django-oidc.git#egg=mozilla-django-oidc`?